account_check_printing_report_dlt103/views/report_check.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <template id="report_check_dlt103">
+            <t t-call="report.html_container">
+                <t t-foreach="docs" t-as="o">
+                    <div class="page">
+                        <div>
+                            <div style="position:absolute; right:24mm; top:25mm;">
+                                <span t-field="o.payment_date"/>
+                            </div>
+                            <div style="position:absolute; top:40mm;">
+                                <span style="position:absolute; left:28mm;"
+                                      t-field="o.partner_id.name"/>
```

Comment:
It seems to me that the partner name is not showing correctly when there are two or more words in the name:

![image](https://cloud.githubusercontent.com/assets/19620251/21549883/c821399a-cdf5-11e6-9211-e0685c8a60c8.png)